ynput/ayon-core
# Attribute definitions: Fix context menu on sequence items

## Changelog Description
Fix issue with sequence item context menu button in FileDef widget.

## Testing notes:
1. Add a `FileDef` allowing sequences to any plugin.
2. Drop into the widget sequence.
3. Click this icon, A menu should show up as expected (with `Split sequence` action).
![Image](https://github.com/user-attachments/assets/118f3636-20b2-4738-bd9a-9bb5498ef94d)

Resolves https://github.com/ynput/ayon-core/issues/1032 .
